release/bump_patch_version.py

```diff
@@ -10,4 +10,4 @@
 with open('../version.txt', 'w') as fd:
     fd.write(f'{version.major}.{version.minor}.{version.micro + 1}\n')
 
-print(f'{version} -> {version.major}.{version.minor}.{version.micro + 1}')
+print(f'{version} -> {version.major}.{version.minor}.{version.micro + 1}')
```

Comment:
This file doesn't have the other updates for handling the `--current-version` arg.